lib/Compilation.js

```diff
@@ -120,8 +120,14 @@ const { getRuntimeKey } = require("./util/runtime");
  */
 
 /**
+ * Plugin instance.
```

Comment:
You can remove these type declarations. They are redundant.

Instead import:

``` js
/** @typedef {import("../declarations/WebpackOptions").WebpackPluginFunction} WebpackPluginFunction */
/** @typedef {import("../declarations/WebpackOptions").WebpackPluginInstance} WebpackPluginInstance */
```

from the schema.

Reply:
I didn't see any WebpackOptions typings inside Compiler so I copied it over but importing is way cleaner 👍